hls4ml/writer/vivado_accelerator_writer.py

```diff
@@ -333,6 +333,13 @@ def write_board_script(self, model):
         filedir = os.path.dirname(os.path.abspath(__file__))
         copyfile(os.path.join(filedir, self.vivado_accelerator_config.get_tcl_file_path()),
                  '{}/design.tcl'.format(model.config.get_output_dir()))
+        if self.vivado_accelerator_config.get_interface() == 'axi_master':
```

Comment:
should this code be run for all `axi_master` interface or just for `arty`? If for all, I guess we need to add `verilog_wrappers` for at least `pynq-z2`

Reply:
otherwise, i get this error:
```
Traceback (most recent call last):
  File "convert.py", line 228, in <module>
    main(args)
  File "convert.py", line 190, in main
    hls_model.compile()
  File "/home/jduarte1/CIFAR10/hls4ml/hls4ml/hls4ml/model/hls_model.py", line 538, in compile
    self.write()
  File "/home/jduarte1/CIFAR10/hls4ml/hls4ml/hls4ml/model/hls_model.py", line 535, in write
    self.config.writer.write_hls(self)
  File "/home/jduarte1/CIFAR10/hls4ml/hls4ml/hls4ml/writer/vivado_accelerator_writer.py", line 553, in write_hls
    self.write_board_script(model)
  File "/home/jduarte1/CIFAR10/hls4ml/hls4ml/hls4ml/writer/vivado_accelerator_writer.py", line 337, in write_board_script
    copytree(os.path.join(filedir, self.vivado_accelerator_config.get_vivado_ip_wrapper_path()),
  File "/home/jduarte1/miniconda3/envs/tiny-mlperf-env/lib/python3.8/shutil.py", line 555, in copytree
    with os.scandir(src) as itr:
FileNotFoundError: [Errno 2] No such file or directory: '/home/jduarte1/CIFAR10/hls4ml/hls4ml/hls4ml/writer/../templates/vivado_accelerator/pynq-z2/verilog_wrappers'
```